Index: ruamel-yaml-00c7352d22d0/setup.py
===================================================================
--- ruamel-yaml-00c7352d22d0.orig/setup.py
+++ ruamel-yaml-00c7352d22d0/setup.py
@@ -235,13 +235,6 @@ class NameSpacePackager(object):
         self.command = None
         self.python_version()
         self._pkg = [None, None]  # required and pre-installable packages
-        if sys.argv[0] == 'setup.py' and sys.argv[1] == 'install' and \
-           '--single-version-externally-managed' not in sys.argv:
-            if os.environ.get('READTHEDOCS', None) == 'True':
-                os.system('pip install .')
-                sys.exit(0)
-            print('error: you have to install with "pip install ."')
-            sys.exit(1)
         # If you only support an extension module on Linux, Windows thinks it
         # is pure. That way you would get pure python .whl files that take
         # precedence for downloading on Linux over source with compilable C
